src/main/java/org/kcctl/command/DescribeConnectorCommand.java

```diff
@@ -59,16 +61,24 @@ public class DescribeConnectorCommand implements Callable<Integer> {
     @Option(names = { "--tasks-config" }, description = "Displays tasks configuration")
     boolean includeTasksConfig;
 
+    @Option(names = { "--output-format" }, description = "Specifies the output format, i.e 'json'")
+    String outputFormat;
```

Comment:
Can we make that an enum? picocli has bespoke support for enum options, e.g. in tab completion. Allowed values would be "text" and "json" for now.